packages/shared-core/src/sdk/documents/automations.ts

```diff
@@ -2,12 +2,12 @@ import { Automation, AutomationTriggerStepId } from "@budibase/types"
 
 export function isRowAction(automation: Automation) {
   const result =
-    automation.definition.trigger.stepId === AutomationTriggerStepId.ROW_ACTION
+    automation.definition.trigger?.stepId === AutomationTriggerStepId.ROW_ACTION
```

Comment:
This is not nullable on the type definition, probably we should check how come this comes null.
Should we check also if `definition` is null?

Reply:
Yea i'm not sure how I got like that, but it will be in the definition for it now regardless. Added a check on the definition property as well just to be sure.